### DISCUSSION TITLE
Restore preference for large CSV files

### DISCUSSION THREAD

**Discussion by fsgeek91** [mechatroner/vscode_rainbow_csv#165] [Q&A] [answered]
When opening a very large CSV file in VS Code, a dialogue appears at the bottom-right asking if I want to disable features or force them to run. I clicked on disable with the option to not ask me again.

How can I re-enable this prompt, or find another way to manually force RBQL to process very large CSV files? I looked int he extension settings and can't find the option.

Thanks for your work - this is a fantastic tool!

**Comment by mechatroner** [comment, marked as answer]
Hi there, and thanks for the feedback! 
Short answer: you can try to adjust `"editor.largeFileOptimizations": false` setting, see: https://stackoverflow.com/questions/53625687/what-is-the-largest-filesize-supported-by-vs-code-syntax-highlighting

Unfortunately, very large files is an unsolvable problem as of now, because VSCode just disables all extension features if the files are bigger than a certain size. There might be also a "mid-range" of file sizes that are fixable with  `editor.largeFileOptimizations`. Another option is to run standalone rbql cli command via `pip install rbql` if you want to use Python version or `npm install rbql` for JS version https://pypi.org/project/rbql/  https://www.npmjs.com/package/rbql

**Reply by fsgeek91** [reply]
That solves it, thanks! I hadn't realised this was a global VSCode setting. I'll also check out the Python implementation.

Cheers
